judges/github-events/github-events.rb

```diff
@@ -240,6 +261,7 @@ def self.fill_up_event(fact, json)
         if fact.all_properties.include?('who') && fact.who != json[:payload][:release][:author][:id]
           Fbe.overwrite(fact, 'who', json[:payload][:release][:author][:id])
         end
+        skip_event(json) if ignored_bot?(fact)
```

Comment:
@danilax86 Is this code really needed here? It seems to me that checking at the beginning (line 133) will be enough.